classes/Product.php

```diff
@@ -7069,7 +7069,7 @@ public function getCoverWs()
     {
         $result = $this->getCover($this->id);
 
-        return $result['id_image'];
+        return $result ? $result['id_image'] : null;
```

Comment:
Please update the PHPDoc :)

Reply:
I don't understand what to add.

Reply:
A few lines before, you have ` * @return int|string`, you should replace it with ` * @return int|string|null` or even ` * @return int|null` if you're sure `$result['id_image']` cannot be a `string`.

Reply:
I see now what you mean. I pushed the change. Null should be legal return value as it is accepted by functions calling getCoverWs